index.html

```diff
@@ -12,34 +12,34 @@
 <body>
     <header class="header-block">
         <button class="drop-btn" onclick="mainModule.showHideMenu()"></button>
-        <h1><img alt="iMolodets rules" src="images/logo/logo-rules-dark.svg" class="title-img"></h1>
+        <h1><a href="#top"><img alt="iMolodets rules" src="images/logo/logo-rules-dark.svg" class="title-img"></a></h1>
     </header>
     <aside>
         <nav class="expand-menu">
             <ul>
                 <li>
                     <a href="#A00" class="list-item" onclick="mainModule.expandMenu(0)">
-                        <span class="list-number">01</span>OBJECT</a>
+                        <span class="list-number">01</span><p>OBJECT</p></a>
                 </li>
```

Comment:
wrapping in p tag isn't valid markup, so let's wrap it with a span tag instead. Please change it in all places.